cmd/semTable.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+    "fmt"
+    "github.com/anaskhan96/soup"
+)
+
+// semTable function - equivalent to your Python code
+func semTable(url string) {
+    fmt.Println("Fetching semesters...")
+    resp := fetchHTML(url)  // Fetch the HTML content
+
+    // Check if the response is empty
+    if resp == "" {
+        fmt.Println("Failed to fetch the HTML content. Exiting.")
+        return
+    }
+    
+    // Parse the HTML content using soup
+    doc := soup.HTMLParse(resp)
+    div := doc.Find("div", "id", "aspect_artifactbrowser_CommunityViewer_div_community-view")
+
+    if div.Error != nil {
+        fmt.Println("No semesters found on the page.")
+        return
+    }
+
+    ul := div.Find("ul")
+    li := ul.FindAll("li")
+
+    if len(li) == 0 {
+        fmt.Println("No semesters found on the page.")
+        return
+    }
+
+    // Display the found items
+    fmt.Printf("No\tSemesters\n")
+    for i, link := range li {
+        a := link.Find("a")
+        if a.Error == nil {
+            span := a.Find("span")
+            if span.Error == nil {
+                fmt.Printf("%d\t%s\n", i+1, span.Text())  // Extract the text from the span element
+            }
+        }
```

Comment:
Don't ignore the error!

```suggestion
        if a.Error != nil {
            fmt.Println(a.Error);
        }
        span := a.Find("span")
        if span.Error != nil {
            fmt.Println(a.Error)
        }

        // Extract the text from the span element.
        fmt.Printf("%d\t%s\n", i+1, span.Text())
```